packages/backend/migrations/20230201133042_create_ledger_accounts_table.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+  return knex.schema.createTable('ledgerAccounts', function (table) {
+    table.uuid('id').notNullable().primary()
+    table.uuid('accountRef').notNullable().unique().index()
+
+    table.uuid('assetId').notNullable()
+    table.foreign('assetId').references('assets.id')
+
+    table
+      .enu('type', [
+        'LIQUIDITY_ASSET',
+        'LIQUIDITY_PEER',
+        'LIQUIDITY_INCOMING',
+        'LIQUIDITY_OUTGOING',
+        'LIQUIDITY_WEB_MONETIZATION',
+        'SETTLEMENT'
+      ])
```

Comment:
Will we have a string enum for postgres, as opposed to the numeric enum we currently have for tigerbeetle?
https://github.com/interledger/rafiki/blob/b7964193d372555edc4862443649085dc7f695bc/packages/backend/src/accounting/tigerbeetle/service.ts#L29-L36

Reply:
Yep, I have this in a separate branch & on the `LedgerAccount` model. Just string enums, exactly as the `type` in the table